README.md

```diff
@@ -27,34 +27,21 @@
 - [External Contributions](#external-contributions)
 - [License](#license)
 
-## Ways to Use the Libraries
-
-Despite the number of modules in this package, there are a number of approaches to using these tools which many be used separately or together in any number of combinations.
-
-* **Precaching, runtime caching and routing**: Use [sw-lib](#sw-lib) to quickly
-  implement [caching strategies](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/) or a route.
-* **Build process**: Generate a service worker or manifest in your build process
-  using [sw-build](#sw-build).
-* **Command line**: Generate a service worker or manifest from the command line.
-* **Go beyond the basics**: Implement more advanced capabilities and more
-  sophisticated use cases using any of the other libraries, which are [described
-  below](#the-libraries).
-
-## Setting Up
+## Installing a module
 
 Each module is installed separately using the command line [as listed below](#the-libraries). To use a particular module:
 
 1. Install the module. For example:
 
-   `npm install --save-dev sw-lib`
+   `npm install --save-dev sw-build`
 2. Copy the module's JavaScript and map files to your serving directory. 
 
-   `cp node_modules/sw-lib/build/* app/`
+   `cp node_modules/sw-build/build/* app/`
```

Comment:
This makes sense for sw-lib, but not for sw-build.

sw-lib you need to import the file in your service worker, sw-build is a node module you'd use at build time.